src/features/common/TreeCounter/TreeCounter.tsx

```diff
@@ -1,44 +1,33 @@
-import CircularProgress, {
+import MuiCircularProgress, {
   CircularProgressProps,
-} from '@material-ui/core/CircularProgress';
-import { createStyles, makeStyles } from '@material-ui/core/styles';
+} from '@mui/material/CircularProgress';
 import React from 'react';
 import treeCounterStyles from './TreeCounter.module.scss';
 import i18next from '../../../../i18n';
 import EditIcon from '../../../../public/assets/images/icons/manageProjects/Pencil';
 import { localizedAbbreviatedNumber } from '../../../utils/getFormattedNumber';
-import { getFormattedNumber } from '../../../utils/getFormattedNumber'
+import { getFormattedNumber } from '../../../utils/getFormattedNumber';
 import themeProperties from '../../../theme/themeProperties';
 import { ThemeContext } from '../../../theme/themeContext';
+import { styled } from '@mui/material/styles';
 const { useTranslation } = i18next;
 
-const useStylesFacebook = makeStyles(() =>
-  createStyles({
-    root: {
-      position: 'relative',
-    },
-    top: {
-      color: '#fff',
-      animationDuration: '550ms',
-    },
-    circle: {
-      strokeLinecap: 'round',
-    },
-  })
-);
+const CircularProgress = styled(MuiCircularProgress)({
+  '&.MuiCircularProgress-root': {
+    color: '#fff',
+    animationDuration: '550ms',
+  },
+  '& > svg > circle': {
+    strokeLinecap: 'round',
+  },
+});
 
 function FacebookCircularProgress(props: CircularProgressProps) {
-  const classes = useStylesFacebook();
-
   return (
-    <div className={classes.root}>
+    <div style={{ position: 'relative' }}>
       <CircularProgress
         variant="determinate"
-        // disableShrink
-        className={classes.top}
-        classes={{
-          circle: classes.circle,
-        }}
+        disableShrink
```

Comment:
I see a console error while viewing this on /profile.

`Warning: Failed prop type: MUI: You have provided the `disableShrink` prop with a variant other than `indeterminate`. This will have no effect.
`

Reply:
![Screenshot 2022-05-09 at 4 33 56 PM](https://user-images.githubusercontent.com/44917347/167397635-b10ef18d-950d-46c3-aef7-2221c447ac7e.png)

Reply:
![image](https://user-images.githubusercontent.com/37466979/167788174-76ff597d-3496-4a01-be80-9e626ed59043.png)

Reply:
Fixed here https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1467/commits/61332318a2202ec4c2e3a942c940a84db33dc3f4